src/components/GameView/GameOverDialog.vue

```diff
@@ -102,20 +148,29 @@ export default {
         });
       });
     },
+    iconFromGameStatus(gameStatus) {
+      switch (gameStatus) {
+        case 'W':
+          return 'mdi-thumb-up-outline';
+        case 'L':
+          return 'mdi-thumb-down-outline';
+        case 'I':
+          return 'mdi-account-clock-outline';
+        case 'D':
+          return 'mdi-handshake-outline';
+      }
+    },
   },
 };
 </script>
 
 <style scoped lang="scss">
 .logo-image {
   height: auto;
-  max-width: 180px;
-
-  // TODO: replace with an actual css variable for breakpoint width
-  // https://github.com/vuetifyjs/vuetify/blob/v3.1.3/packages/vuetify/src/styles/settings/_variables.scss#L95
-  @media(min-width: 601px) {
-    max-width: 300px;
-  }
+  max-width: 90px;
```

Comment:
let's add max-height as well so the stalemate image isn't super tall
```suggestion
  max-width: 90px;
  max-height: 90px;
```